src/PrestaShopBundle/Controller/Admin/Improve/Design/ThemeController.php

```diff
@@ -220,6 +222,9 @@ public function importAction(Request $request)
 
                 $this->getCommandBus()->handle(new ImportThemeCommand($importSource));
 
+                $event = new ThemeManagementEvent('montheme');
+                $this->get('event_dispatcher')->dispatch($event, ThemeManagementEvent::INSTALL);
```

Comment:
IMO we should add a dispatcher interface to dispatch event inside the command with an adapter based on symfony dispatcher

Reply:
Agreed, this should be triggered in `ImportThemeHandler` or actually even in `ThemeManager::install` this way you're sure the event is always triggered even when the command is not used